packages/build-tools/src/utils/expoUpdates.ts

```diff
@@ -152,8 +157,9 @@ export async function configureExpoUpdatesIfInstalledAsync(ctx: BuildContext<Job
     );
   }
 
-  if (ctx.job.updates?.channel) {
-    await configureEASExpoUpdatesAsync(ctx);
+  const channel = ctx.job.updates?.channel ?? ctx.metadata?.buildProfile;
+  if (ctx.appConfig.updates?.url && channel) {
+    await configureEASExpoUpdatesAsync(ctx, channel);
```

Comment:
@brentvatne Are you sure we want to make an override server-side? The breaking change is the same as I described in eas-cli PR, but the change here will affect everyone, not only users that updated to the latest cli.

Reply:
i'll get back to you on this, i mentioned it to @jonsamp and he is going to think about it